compiler/src/main/java/org/robovm/compiler/config/Config.java

```diff
@@ -1361,7 +1396,12 @@ public void addPluginArgument(String argName) {
             }
             config.pluginArguments.add(argName);
         }
-
+        
+        // TODO: Are we disobeying any rules by allowing config to be fetched?  All I want is the targetBinary.
```

Comment:
Hi, 

Thanks, I will address this! 

Best Regards 

Chris Fogelklou, 
Applaud Apps (Applicaudia) 
Sent from my mobile, excuse the brevity. 

-------- Original Message --------
From:Mario Zechner <notifications@github.com>
Sent:Sat, 20 Jun 2015 13:02:46 +0200
To:robovm/robovm <robovm@noreply.github.com>
Cc:cfogelklou <chris.fogelklou@applaudapps.net>
Subject:Re: [robovm] Feat/chfo/mergedylib6 compiler (#1020)

>In compiler/src/main/java/org/robovm/compiler/config/Config.java:
>
>> @@ -1361,7 +1396,12 @@ public void addPluginArgument(String argName) { > } > config.pluginArguments.add(argName); > } > - > + > + // TODO: Are we disobeying any rules by allowing config to be fetched? All I want is the targetBinary. 
>
>I think i'd be better to have a getter in the builder. We do something similar for plugins imo.
>
>—
>Reply to this email directly or view it on GitHub.￼
>